cpp/src/Ice/SSL/SchannelTransceiverI.cpp

```diff
@@ -113,7 +109,7 @@ Schannel::TransceiverI::getNativeInfo()
 }
 
 IceInternal::SocketOperation
-Schannel::TransceiverI::sslHandshake()
+Schannel::TransceiverI::sslHandshake(SecBuffer* initialBuffer)
```

Comment:
this initial buffer is used when a TLS renegotiation occurs during the handshake. When using TLS 1.3 seems to happen all the time on Windows.